.github/workflows/on-nextjs-release-publish.yml

```diff
@@ -33,3 +41,34 @@ jobs:
     uses: ./.github/workflows/upload-nextjs-integration-test-results.yml
     with:
       is_main_branch: false
+
+  post_to_slack:
+    needs: [next_js_integration]
+    name: Post results to Slack
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download summary.md artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: test-results
+      - name: Check if summary file was generated
+        id: summary_check
+        run: |
+          if stat test-summary.md; then
+            echo "should_continue=true" >> $GITHUB_OUTPUT
+          else
+            echo "should_continue=false" >> $GITHUB_OUTPUT
+          fi
+      - name: Build payload
+        if: steps.summary_check.outputs.should_continue == 'true'
+        run: |
+          SUMMARY_MD=$(cat test-summary.md)
+
+          jq --null-input --arg text "$SUMMARY_MD" '{"text": $text}' > payload.json
```

Comment:
if we need json payload, we can actually make test-summary.md itself to be a json instead. Should we do it (in following PR)?